src/WizardLayout/WizardLayout.tsx

```diff
@@ -112,7 +112,7 @@ export default class WizardLayout extends React.PureComponent<Props> {
           {headerImg && <div className={cssClass.HEADER_IMG}>{headerImg}</div>}
           <div>
             <p className={cssClass.HEADER_TITLE}>{title}</p>
-            {subtitle && <p className={cssClass.HEADER_SUBTITLE}>{subtitle}</p>}
+            {subtitle && <div className={cssClass.HEADER_SUBTITLE}>{subtitle}</div>}
```

Comment:
> major web browsers will render a <p> tag with margin above and below the paragraph. A <div> tag will be rendered without any margin at all

Can we conditionally render p and div tags depending on if the subtitle is a string so that there's no UI regression?